fix X-Acelle-Customer-Id default value

jap opened the pull request:
This prevents tons of lookups for acelle with  customer id '0'

jap committed:
fix X-Acelle-Customer-Id default value

This prevents tons of lookups for acelle with  customer id '0'

bigio reviewed (changes requested):
If customer is 0 I think the sub should return, so line 668 should be changed accordingly.

jap committed:
Update Esp.pm

If the customer-id header contains a false-y value (empty string, 0, etc) or is not defined, return early.
